projects/igniteui-angular/src/lib/services/excel/worksheet-data.ts

```diff
@@ -11,8 +11,8 @@ export class WorksheetData {
     private _isSpecialData: boolean;
 
     constructor(private _data: any[], public options: IgxExcelExporterOptions, public indexOfLastPinnedColumn,
-                public sort: any, public isTreeGridData = false) {
-        this.initializeData();
+                public sort: any, public isTreeGridData = false, public columnWidths: number[]) {
```

Comment:
Change to private _columnWidths and remove parameter from initializeData method.